OpenQA/Isotovideo/Utils.pm

```diff
@@ -195,11 +195,11 @@ sub clone_git ($local_path, $clone_url, $clone_depth, $branch, $dir, $dir_variab
         $clone_depth *= 2;
         @out = qx[git -C $local_path fetch --progress --depth=$clone_depth 2>&1];
         $handle_output->($?, @out);
-        die "Could not find '$branch' in complete history in cloned Git repository '$dir'" if grep /remote: Total 0/, @out;
+        die "Could not find '$branch' in complete history in cloned Git repository \"$dir\"" if grep /remote: Total 0/, @out;
     }
     @out = qx{git -C $local_path checkout $branch};
     bmwqemu::diag "@out" if @out;
-    die "Unable to checkout branch '$branch' in cloned Git repository '$dir'" unless $? == 0;
+    die "Unable to checkout branch '$branch' in cloned Git repository $dir" unless $? == 0;
```

Comment:
```suggestion
    die "Unable to checkout branch '$branch' in cloned Git repository \"$dir\"" unless $? == 0;
```

or is there a reason for exception?

Reply:
No exception. I was running jobs to test @perlpunk PR. I will fix

Reply:
Done https://github.com/os-autoinst/os-autoinst/commit/ea085328ccc8838cf86cc8062810ac1da49a1676